flent/scripts/tc_iterate.sh

```diff
@@ -23,7 +21,7 @@ buffer=""
 
 
 command_string=$(cat <<EOF
-which tc_iterate >/dev/null && exec tc_iterate $buffer -i $interface -c $count -I $interval -C $command;
+which tc_iterate >/dev/null && exec tc_iterate $buffer -i $interface -l $length -I $interval -C $command;
```

Comment:
OK, so one thing that comes to mind as a way of keeping backwards compatibility, is that instead of unconditionally exec'ing when the binary exists, we do a regular call to the binary, and exit the script on success. That way, if it's an old binary without the `-l` option, we will fall back to the script. The only potential problem with this is that if the binary runs, but still exits non-zero, we'll get duplicate results. This could happen if it explicitly killed; however, it seems that that results in a different exit code, so maybe if we only react to the return value in `usage()` (255), that would work?

Reply:
another option could be to check the output of the binary when called with `--help` (or similar) and basically grep if `-l` or `-c` is in the output. This way we could distinguish between the binaries.
Or we just keep `-c` and calculate the `count` in the script instead of in the C-code. This would be probably the minimal fix. But either way, depending on your preference I am happy to implement it :)

Reply:
Oh yeah, that's a good point actually: why is this change needed at all? You're basically just moving the same calculation from python to C?

Reply:
I thought doing it in C makes it easier if one (like me) wants to directly execute the C-binary. And then it is just more comfortable to pass a length parameter instead of deriving the count from the duration. But if preferred, then I would just add the calculation to the python code (or in the bash script?)

Reply:
Well, :D you are correct, sorry I misread the code. Will close this PR then.